strategy/measuring-impact.md

```diff
@@ -0,0 +1,148 @@
+| -----------: | ---- | :-------- |
+| 1 | a. | Access statistics for DCP data releases are higher than access statistics for the “live” DCP view |
+| 2 | a. | Access statistics for data specific subsets are higher than access statistics for the “live” DCP view |
+| 3 | a. | Access statistics for migrated datasets in the DCP release are higher than access statistics for unmigrated data in the “live” DCP view |
+|   | b. | Frequent requests to curate historical datasets that do not conform to the latest standards, and are therefore missing from the latest releases, are received by DCP helpdesk |
+|   | c. | A number of historical datasets perceived in the community as having high value, but that are non-conformant to latest standards, are disproportionately heavily accessed from previous data releases |
+| 4 | a. | Access statistics for reprocessed DCP data are higher than access for the original data |
+| 5 | a. | Access statistics for previous data releases is initially higher than access statistics for the latest release, but access diminishes over time |
+
+
+### Aim 4
+
+**Build alignment amongst the Human Cell Atlas community around a core set of computational methods and analyses.**
+
+#### Value Assumptions
+
+The Human Cell Atlas community are experimenting with a variety of methods of data processing and analysis, and it is currently unclear which methods are “best”.
+
+Scientists expect the DCP to provide information about the most commonly used methods for processing and analysis of cellular resolution data.
+
+Until convergence emerges, scientists will process data in a variety of ways, and are likely to publish on the accuracy of methods, including those in the DCP, significantly changing the “state of the art”.
+
+Scientists expect that all data generated by the most commonly utilised technologies can be analysed by DCP processing pipelines
+
+Analysing integrated data across experiments is most easily accomplished when technical variation is minimized or controlled for. When performing such integrative analyses, scientists will expect DCP data to be produced from “harmonized” data processing pipelines.
+
+Scientists expect that the latest and greatest analysis methods are quickly available within the DCP, and if the methods they need are not available, will work with the DCP analysis community to contribute them.
+
+Scientists expect to benchmark datasets by processing and analysing them with computational methods within or alongside the DCP and comparing them to results from other methods.
+
+#### Indicators for Success
+
+| Specific aim | Item | Indicator |
+| -----------: | ---- | :-------- |
+| 1 | a. | DCP technology reports are highly accessed by a variety of users |
+|   | b. | DCP technology reports receive positive feedback from the wider single cell community |
+| 2 | a. | The number of datasets that are generated and deposited in the DCP, but that cannot be analysed by DCP pipelines, is very low |
```

Comment:
Maybe mention of a workflow sharing registry like dockstore.org fits better here than above?